proto/finschia/or/settlement/v1/tx.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+package finschia.or.settlement.v1;
+
+import "finschia/or/settlement/v1/settlement.proto";
+
+option go_package = "github.com/Finschia/finschia-sdk/x/or/settlement/types";
+
+// Msg defines the Msg service.
+service Msg {
+  rpc StartChallenge(MsgStartChallenge) returns (MsgStartChallengeResponse);
+  rpc NsectChallenge(MsgNsectChallenge) returns (MsgNsectChallengeResponse);
+  rpc FinishChallenge(MsgFinishChallenge) returns (MsgFinishChallengeResponse);
+}
+
+message MsgStartChallenge {
+  string from         = 1;
+  string to           = 2;
+  string rollup_name  = 3;
+  int64  block_height = 4;
+  uint64 step_count   = 5;
+}
+
+message MsgStartChallengeResponse {
+  string challenge_id = 1;
+}
+
+message MsgNsectChallenge {
+  string         from         = 1;
+  string         challenge_id = 2;
+  repeated bytes state_hashes = 3;
```

Comment:
Are you planning to get this from the DA layer's scc batch in the future?

Reply:
No. `state_hashes` means vm state hashes. The number of this value is about 10 million per block so these huge data can't stored in DA.

Reply:
So you're exchanging 10 million state_hashes with the L2 node?
Also, in that case, how does the user L2 node calculate state_hashes and send msg?